app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -10,14 +10,23 @@ HTMLDocument.prototype.ready = new Promise((resolve) => {
 
 // If localStorage.getItem('shouldRedirectToOnboarding') is not set, i.e. null, that means we should redirect.
 function redirectableLocation() {
-  return (
-    window.location.pathname !== '/onboarding' &&
-    window.location.pathname !== '/signout_confirm' &&
-    window.location.pathname !== '/privacy' &&
-    window.location.pathname !== '/code-of-conduct' &&
-    window.location.pathname !== '/terms' &&
-    window.location.pathname !== '/admin/creator_settings/new'
-  );
+  return ![
+    '/onboarding',
+    '/signout_confirm',
+    '/privacy',
+    '/admin/creator_settings/new',
+  ].includes(window.location.pathname);
+}
+
+function redirectableCreatorOnboardingLocation() {
+  return ![
+    '/onboarding',
```

Comment:
Question: could it make sense to reuse the other function here to avoid drift between them (e.g. `return redirectableLocation() && !['/code-of-conduct', 'term'].includes(window.location.pathname);`) or are we uncomfortable with this level of coupling?

Reply:
Great suggestion, @citizen428! I've updated this.